editor/src/dev/load_project.clj

```diff
@@ -0,0 +1,84 @@
+;; Copyright 2020-2024 The Defold Foundation
+;; Copyright 2014-2020 King
+;; Copyright 2009-2014 Ragnar Svensson, Christian Murray
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;;
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;;
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+(ns load-project
+  (:require [dev :as dev]
+            [dynamo.graph :as g]
+            [editor.defold-project :as project]
+            [editor.editor-extensions :as extensions]
+            [editor.shared-editor-settings :as shared-editor-settings]
+            [editor.workspace :as workspace]
+            [integration.test-util :as test-util]
+            [internal.graph.types]
+            [internal.system :as is]
+            [util.debug-util :as du]))
```

Comment:
This file is more of a playground template for messing around with a loaded project graph without the UI. It's useful during memory profiling since there aren't any UI or background tasks running that perform allocations in the background.